src/routes/auth.js

```diff
@@ -18,7 +21,9 @@ const {
 
 const router = express.Router()
 const ERROR_MAP = {
-	[ refreshTokenIsInvalidError.name ] : 401
+	[ refreshTokenIsInvalidError.name ] : 401,
+	[ inviteEmailDoesNotExistError.name ] : 403,
+	[ userAlreadyActivatedError.name ] : 400
```

Comment:
@itzgeebee,  so for our purposes, `400` will actually be related to the request being malformed or not fitting our schema. Reference [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400)

in this case, the request is actually correct but the server can't process it so let's go with `422`. Reference [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422)